Assets/Scripts/World/Sun.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using UnityEngine.Experimental.Rendering.LWRP;
+using Utils;
+
+namespace World
+{
+    public class Sun : MonoBehaviour
+    {
+        [SerializeField] private Transform rotationPoint;
+        [SerializeField] private Light2D globalLight;
+        [SerializeField] private float dayLengthInS;
```

Comment:
Würde es nur dayLength nennen und evtl. noch nen Tooltip für In Seconds erstellen